core/src/main/java/io/questdb/std/Chars.java

```diff
@@ -120,30 +120,34 @@ public static int compareDescending(CharSequence l, CharSequence r) {
     }
 
     public static boolean contains(CharSequence sequence, CharSequence term) {
+        return contains(sequence, 0, sequence.length(), term) != -1;
+    }
+
+    public static int contains(CharSequence sequence, int sequenceLo, int sequenceHi, CharSequence term) {
```

Comment:
minor: this usually expected to have name `indexOf` not `contains`

Reply:
fixed here: https://github.com/questdb/questdb/pull/2346/commits/0f980d15d9c5d50cc9c1db2a216fba39b0a3ef81